docs/.gitignore

```diff
@@ -0,0 +1,12 @@
+.DS_Store
```

Comment:
DS_Store is normally the responsibility of your person .gitignore.

Reply:
It's already in the main `.gitignore` 😁